src/doctor.ts

```diff
@@ -251,4 +251,12 @@ export class Doctor implements SfDoctor {
   private writeFile(stream: fs.WriteStream, contents: string): Promise<boolean> {
     return Promise.resolve(stream.write(contents));
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  private createOutoutDor(fullPath: string): void {
```

Comment:
Is this supposed to be, `createOutputDir`?

Reply:
Wow did I really type that or can I blame ChatGPT :)